metadata-ingestion/src/datahub/ingestion/source/superset.py

```diff
@@ -251,15 +256,20 @@ def emit_dashboard_mces(self) -> Iterable[MetadataWorkUnit]:
 
         while current_dashboard_page * PAGE_SIZE <= total_dashboards:
             dashboard_response = self.session.get(
-                f"{self.config.connect_uri}/api/v1/dashboard",
+                f"{self.config.connect_uri}/api/v1/dashboard/",
```

Comment:
???

Reply:
Yep - that's what caused the incompatibility with superset v2. I tested with v1 as well and that seems to still work